rpcs3/Emu/SysCalls/Modules/cellFont.cpp

```diff
@@ -566,12 +632,10 @@ s32 cellFontGetCharGlyphMetricsVertical()
 
 Module cellFont("cellFont", []()
 {
-	s_fontInternalInstance = new CCellFontInternal();
+	s_fontInternalInstance = new CellFontInternal();
 
 	cellFont.on_stop = []()
 	{
-		// s_fontInternalInstance->m_bInitialized = false;
-		// s_fontInternalInstance->m_bFontGcmInitialized = false;
 		delete s_fontInternalInstance;
```

Comment:
Need to ban new/delete usage in rpcs3... With std::unique_ptr, using new/delete is pointless.